ui/app/components/sidebar/nav/cluster.js

```diff
@@ -15,4 +15,9 @@ export default class SidebarNavClusterComponent extends Component {
   get cluster() {
     return this.currentCluster.cluster;
   }
+
+  get isRootNamespace() {
+    // should only return true if we're in the true root namespace
```

Comment:
ty for the comment!

Reply:
I'm still getting up to speed with the `chroot` situation - does true "root" mean `root` root? 

Does `this.currentCluster.hasChrootNamespace` mean the user has a configured root namespace and so they will not have `root` as the base/parent namespace?

Reply:
`hasChrootNamespace` means the Vault operator has set `chroot_namespace` to some value on the config of the given listener, and so the user does not have access to "true root" (which yes, is `root` namespace). So, even if the UI is in its top-most namespace, if `chroot_namespace` is configured it is not the true root. 

Does that answer your question?

Reply:
Yes! I think so - I wanted to clarify that setting the `chroot_namespace` means that you are configuring the top namespace and so therefore `root` will NOT be a possible/accessible. 

Thank you!